en/rocky/8/admin_guide/01-presentation.md

```diff
@@ -0,0 +1,299 @@
+
+* **2004**: Ubuntu is created by the Canonical company (Mark Shuttleworth). It is based on Debian, but includes free and proprietary software.
+
+* **2021**: Birth of Rocky Linux, based on RedHat distribution.
+
+### Market share
+
+<!--
+TODO: graphics with marketshare for servers and pc.
+-->
+Linux is still not well known by the general public, even though they use it regularly. Indeed, Linux is hidden in **smartphones**, **televisions**, **internet boxes**, etc. Almost **70% of the web pages** served in the world are served by a Linux or UNIX server!
+
+Linux equips a little more than **3% of personal computers** but more than **82% of smartphones**. **Android** being an operating system whose kernel is a Linux.
+<!-- TODO: review those stats -->
+
+Linux equips 100% of the 500 supercomputers since 2018. A supercomputer is a computer designed to achieve the highest possible performance with the techniques known at the time of its design, especially with regard to computing speed.
+
+### Architectural design
+
+* The **kernel** is the first software component.
+ * It is the heart of the UNIX system.
+ * It manages the hardware resources of the system.
+ * The other software components must go through it to access the hardware.
+* The **shell** is a utility that interprets user commands and ensures their execution.
+ * Main shells: Bourne shell, C shell, Korn shell and Bourne-Again shell (bash).
+* Applications are user programs such as :
+ * Internet browser ;
+ * the word processor ;
+ * ...
+
+#### Multitask
+
+Linux belongs to the family of time-sharing operating systems. It shares process time between several programs, switching from one to another in a transparent way for the user. This implies:
+
+* simultaneous execution of several programs;
+* distribution of CPU time by the scheduler;
+* reduction of problems due to a failed application;
+* reduced performance when there are too many programs running.
+
+#### Multi-users
+
+The purpose of Multics was to allow several users to work from several terminals (screen and keyboard) on a single computer (very expensive at the time). Linux, which is inspired by this operating system, has kept this ability to work with several users simultaneously and independently, each one having its own user account, memory space and access rights to files and software.
+
+#### Multiprocesseur
+
+Linux is able to work with multi-processor computers or with multi-core processors.
+
+#### Multiprocessor
+
+Linux is written in a high-level language that can be adapted to different types of platforms during compilation. It therefore runs on :
+
+* home computers (PC or laptop);
+* servers (data, applications,...);
+* portable computers (smartphones or tablets)
+* embedded systems (car computer);
+* active network elements (routers, switches)
+* household appliances (TVs, refrigerators,...).
+
```

Comment:
I believe that you have the heading in here twice. The section should be, If I'm getting this correctly:

"#### Multiprocessor

Linux is able to work with multi-processor computers or with multi-core processors.

Linux is written in a high-level language .... " and then the rest of that section. 

Also is it Multi-processor, multiprocessor, or multi processor. I believe that latter is correct? Probably would be the same for multi core. Whatever it is, it should be the same in the title as it is in the body of the text.

Reply:
It's my mistake.
It should be "multi-plateform".